pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -205,6 +205,12 @@ func (c *Cluster) Start() error {
 
 	// configure the mgr modules
 	c.configureModules(daemonIDs)
+
+	// Fetch performance counters from the prometheus exporter as ceph-exporter is disabled for 4.13
+	err = monStore.Set("mgr", "mgr/prometheus/exclude_perf_counters", false)
+	if err != nil {
+		return errors.Wrapf(err, "failed to enable exclude_perf_counters")
```

Comment:
nit: the err message must be failed to `disable` in this case or more dev/debug friendly something like `"failed to enable prometheus mgr module perf counters"`

Reply:
Okay, but I feel like this message is more about the module option enabling/disabling